azimuth/dataset_split_manager.py

```diff
@@ -113,6 +122,11 @@ def get_dataset_split(self, table_key: Optional[PredictionTableKey] = None) -> D
         Returns:
             Dataset with predictions if available.
         """
+        current_last_update = self._base_dataset_split_last_update
+        newest_base_ds, last_update = self.load_latest_cache(self._save_path, current_last_update)
+        if newest_base_ds:
+            self._base_dataset_split = newest_base_ds
```

Comment:
This is "newest" in the sense of "latest", tight? I say we keep it consistent.
```suggestion
        latest_base_ds, last_update = self.load_latest_cache(self._save_path, current_last_update)
        if latest_base_ds:
            self._base_dataset_split = latest_base_ds
```

Reply:
Good point!